server/game_service.py

```diff
@@ -117,6 +118,9 @@ def __init__(self, player_service, game_stats_service):
             # meh meh
             self.ladder_service = LadderService(self)
 
+            # meh meh meh, TODO: pls explain why I am doing this here
+            self.team_matchmaking_service = TeamMatchmakingService()
```

Comment:
We should create services in `server.py`. There might be a better way to do this in the future, but for now putting them all in `server.py` at least ensures that all service creation happens in one place. 

Definitely do not create it in `update_data` here because that will cause the service to be re-created about every 10 minutes. (I told Sheeo already, but I can move LadderService to `server.py` once I find some time, so you don't need to worry about that)